src/routes/forms.test.js

```diff
@@ -427,7 +425,7 @@ describe('Forms route', () => {
     )
 
     test('Testing GET /forms/{id} route with an id that is not found returns 404 Not found', async () => {
-      jest.mocked(getForm).mockResolvedValue(undefined)
+      jest.mocked(getForm).mockRejectedValue(Boom.notFound())
```

Comment:
If we update the form metadata methods to throw (like we've done for definitions) then we'll need to update this mock here to reject from `formMetadata.get()` instead

```suggestion
      jest.mocked(formMetadata.get).mockRejectedValue(Boom.notFound())
```

Reply:
Otherwise you're just testing the mock though?

Reply:
E.g. Our contract with `getForm()` includes:

1. Returning form metadata
2. Throwing an HTTP 404 error
3. Ensuring any HTTP errors add "Form with ID XXX…" text

By mocking `getForm()` (rather than the metadata call) we lose coverage for 3)